tactile_calibration/tactile_state_calibrator/src/tactile_state_calibrator.cpp

```diff
@@ -325,14 +323,8 @@ TactileStateCalibrator::~TactileStateCalibrator()
   // unregister
   tactile_sub_.shutdown();
   // cleanup
-  if (calibs_.size())
-  {
-    for (auto p : calibs_)
-    {
-      delete p;
```

Comment:
As the same calib pointer is stored for several taxels, this delete will multiply free the memory!

Reply:
thanks for spotting that double free.
what happened, is an incomplete optimization session. At first the calibration were duplicates.